modules/restore-replace-stopped-etcd-member.adoc

```diff
@@ -260,13 +260,15 @@ metadata:
   providerID: aws:///us-east-1a/i-0fdb85790d76d0c3f
 ----
 
-.. Delete the machine of the unhealthy member:
+.. Delete the `BareMetalHost` object and the machine of the unhealthy member:
 +
 [source,terminal]
 ----
-$ oc delete machine -n openshift-machine-api clustername-8qw5l-master-0 <1>
+$ oc delete bmh -n openshift-machine-api <host_name> <1>
+$ oc delete machine -n openshift-machine-api clustername-8qw5l-master-0 <2>
 ----
-<1> Specify the name of the control plane machine for the unhealthy node.
+<1> Specify the name of the bare-metal host for the unhealthy node.
+<2> Specify the name of the control plane machine for the unhealthy node.
 
```

Comment:
In step 7 and 8 if another [section](https://docs.openshift.com/container-platform/4.12/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-baremetal-etcd-member_replacing-unhealthy-etcd-member), the procedure deletes the machine first, then the BMH. Is this incorrect? Should the BMH be delete first?